packages/runtime/test/src/interpreter/opcode-list.ts

```diff
@@ -1034,8 +1034,10 @@ describe("Teal Opcodes", function () {
 
 	describe("Ed25519verify", function () {
 		const stack = new Stack<StackElem>();
+		const interpreter = new Interpreter();
 
-		it("should push 1 to stack if signature is valid", function () {
+		//these two tests should be removed or changed
```

Comment:
I think we could remove this two following tests since they cannot be a unit tests anymore. In order to test the `ed25519` opcode we need access to the entire program code through `interpreter`

Reply:
I agree with you but then you should move the test from `interpreter.ts` to this file

Reply:
moved it